ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -289,13 +310,18 @@ protected int getPartitions(KafkaTopicClient kafkaTopicClient) {
   private void collectAggregateArgExpressions(
       final List<Expression> expressions,
       final List<Pair<String, Expression>> aggArgExpansionList,
-      final Map<String, Integer> expressionNames
+      final Map<String, Integer> expressionNames,
```

Comment:
call this internalNameToIndexMap ? ditto for the usage of this name in other parts of the code